controllers/pwa.js

```diff
@@ -58,9 +60,14 @@ router.get('/', (req, res, next) => {
         showScore: sortOrder === 'score',
         pwaCount: pwaCount,
         startPwa: start + 1,
-        endPwa: Math.min(pwaCount, end)
+        endPwa: Math.min(pwaCount, end),
+        mainPage: true
       });
-      res.render('pwas/list.hbs', arg);
+      if (contentOnly) {
```

Comment:
I think we can make this more flexible. The idea is to have different templates for the same content. 
I was playing around with this idea. Check this commit: https://github.com/andreban/gulliver/commit/3c2bd7d33ecf23bd01d9a1a8c71a447eefb89cd5